pennylane/optimize/adaptive.py

```diff
@@ -204,9 +205,7 @@ def step_and_cost(self, circuit, operator_pool, drain_pool=False, params_zero=Tr
         if params_zero:
             params = np.zeros(len(selected_gates))
         else:
-            params = np.array(
-                [gate.parameters[0]._value for gate in selected_gates], requires_grad=True
-            )
+            params = np.array([gate.parameters[0] for gate in selected_gates], requires_grad=True)
```

Comment:
How come this was changed?

Reply:
It was failing a test after adding `g = g.__copy__()` with:

`FAILED test_adaptive.py::test_qubit_rotation[qubit_rotation_circuit] - AttributeError: 'tensor' object has no attribute '_value'`

Before copying the gate, the type of the added gate parameter was: `autograd.numpy.numpy_boxes.ArrayBox`. Now that we copy the gate, the type of the parameter is `pennylane.numpy.tensor.tensor` and `._value` is not needed anymore.